src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -28,274 +43,316 @@ function setupTest(
-          if (!mobile) {
             test(
-              'moves focus to close button when panel content is changed using second button',
+              'focuses split panel preferences button when its position changes from bottom to side',
               setupTest(
                 async page => {
-                  await page.click(
-                    wrapper.findContentRegion().findButton('[data-testid="open-drawer-button"]').toSelector()
-                  );
-                  await page.click(
-                    wrapper.findContentRegion().findButton('[data-testid="open-drawer-button-2"]').toSelector()
-                  );
-                  await page.keys('Tab');
-                  await expect(page.isFocused(wrapper.findActiveDrawerCloseButton().toSelector())).resolves.toBe(true);
+                  await page.click(wrapper.findSplitPanel().findOpenButton().toSelector());
+                  await page.click(wrapper.findSplitPanel().findPreferencesButton().toSelector());
+                  await page.keys(['Tab', 'Right', 'Tab', 'Tab', 'Enter']);
+                  await expect(
+                    page.isFocused(wrapper.findSplitPanel().findPreferencesButton().toSelector())
+                  ).resolves.toBe(true);
                 },
-                { pageName: 'with-drawers', visualRefresh, mobile }
+                { pageName: 'with-split-panel', visualRefresh, mobile, splitPanelPosition: 'bottom' }
               )
             );
+
             test(
-              'moves focus back to last opened button when panel is closed',
+              'focuses split panel preferences button when its position changes from side to bottom',
               setupTest(
                 async page => {
-                  await page.click(
-                    wrapper.findContentRegion().findButton('[data-testid="open-drawer-button"]').toSelector()
-                  );
-                  await page.click(
-                    wrapper.findContentRegion().findButton('[data-testid="open-drawer-button-2"]').toSelector()
-                  );
-
-                  await page.click(wrapper.findActiveDrawerCloseButton().toSelector());
+                  await page.click(wrapper.findSplitPanel().findOpenButton().toSelector());
+                  await page.click(wrapper.findSplitPanel().findPreferencesButton().toSelector());
+                  await page.keys(['Tab', 'Left', 'Tab', 'Tab', 'Enter']);
                   await expect(
-                    page.isFocused(
-                      wrapper.findContentRegion().findButton('[data-testid="open-drawer-button-2"]').toSelector()
-                    )
+                    page.isFocused(wrapper.findSplitPanel().findPreferencesButton().toSelector())
                   ).resolves.toBe(true);
                 },
-                { pageName: 'with-drawers', visualRefresh, mobile }
+                { pageName: 'with-split-panel', visualRefresh, mobile, splitPanelPosition: 'side' }
```

Comment:
Missing `appLayoutWidget` flag here. Also, see the comment TBD how to avoid manually passing it on every test